starboard/linux/x64x11/toolchain/BUILD.gn

```diff
@@ -13,8 +13,27 @@
 # limitations under the License.
 
 import("//build/config/clang/clang.gni")
+import("//starboard/evergreen/shared/toolchain/toolchain.gni")
 import("//starboard/shared/toolchain/overridable_gcc_toolchain.gni")
 
+gcc_toolchain("cobalt") {
+  cc = evergreen_target_cc
+  cxx = evergreen_target_cxx
+  ld = evergreen_target_ld
```

Comment:
FYI @y4vor 

So for the PS5 implementation the cobalt toolchain linker and the starboard toolchain linker are the same.
If linking using the evergreen linker works, that's great or do we want to specify the linux-x64x11 platform linker here ?

Reply:
Let's take this conversation out of the PR review